src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/FeatureFlagController.php

```diff
@@ -93,7 +93,7 @@ public function indexAction(Request $request): Response
             'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
             'enableSidebar' => true,
             'layoutHeaderToolbarBtn' => [],
-            'layoutTitle' => $this->trans('New & Experimental Features', 'Admin.Advparameters.Feature'),
+            'layoutTitle' => $this->trans('New & experimental features', 'Admin.Navigation.Menu'),
```

Comment:
If you change layout title of pages accessible from the menu you should also change the default install data, or the many and the page content won't match

Reply:
It changes the title only, not the menu. No install or upgrade is needed.